### DISCUSSION TITLE
AAVE V1 didn't upgrade to V2 it Seems, Workarounds?

### DISCUSSION THREAD

**Discussion by jomarip** [WalletConnect/walletconnect-monorepo#3585] [Developer Support] [answered]
Good Day,

I was trying to investigate the status of funds that I have on Aave V1 from my Safe Multisig. It seems that they may not have updated their wallet connect and would appreciate a workaround. I don't even need to execution the transactions, but understand the status of the accounts. 

This information is from the console log
```

WebSocket connection to 'wss://aave.bridge.walletconnect.org/' failed: 
value @ 5.ab4b95fb.chunk.js:2
```

**Comment by glitch-txs** [comment, marked as answer]
Hi, there are no workarounds for this they need to update to v2 internally, thanks for reporting 🙏
